bin/spring

```diff
@@ -46,4 +46,5 @@ end
 lib = File.expand_path("../../lib", __FILE__)
 $LOAD_PATH.unshift lib unless $LOAD_PATH.include?(lib) # enable local development
 require 'spring/client'
+Spring.verbose_output = false if $0.include? 'spring' # don't output running spring status if command has `spring` in it
```

Comment:
> how about: Spring.verbose_output = $0.include?('spring')

I think you meant `Spring.verbose_output = !$0.include?('spring')`
It is not going to work it will set `Spring.verbose_output` to `true` when command includes`spring`, even when user has `Spring.verbose_output = false` defined in `config/spring.rb` (or `~/.spring.rb`).